README.md

```diff
@@ -69,9 +79,10 @@ Following parameters is possible to configure from the `screen` -> `variant` fie
 | ctaBoxDescription     | Text          | `"ctaBoxDescription":"Skan koden og indsend dit bidrag"` |
 | ctaBoxImage           | Url           | `"https://pathtoimagesource.io/ctaimage.svg"` |
 | ctaBoxBackgroundColor | Hex format    | `"ctaBoxBackgroundColor":"#fff"` |
+| colors                | array of hex  | `["#ee7752", "#e73c7e", "#23a6d5", "#23d5ab"]` |
```

Comment:
colors seem a bit broad as a name in this context? If we only use them in relation to the skeleton, maybe we could make this more obvious with a name like skeletonColors?

Reply:
will change this too :)